pkg/process/checks/process.go

```diff
@@ -242,6 +252,13 @@ func (p *ProcessCheck) RunWithOptions(cfg *config.AgentConfig, nextGroupID func(
 	return nil, errors.New("invalid run options for check")
 }
 
+func (p *ProcessCheck) processDiscoveryHintCheck(hints *[]model.CollectorProcHint) {
```

Comment:
Will this fail since we're not adding appending a `hintUnspecified` ([code](https://github.com/DataDog/agent-payload/pull/205/files#diff-69b34851fbf6263111121260c46c57084a1fe6e34e9b750a5a673694f2dfe1b0R35))? My understanding was that the length of the hints array needs to match the number of procs in a `CollectorProc`

Reply:
What would fail exactly? Each collectorProc should have at most 2 hints associated with it: `[hintUnspecified, hintProcessDiscovery]`